controllers/Backend.php

```diff
@@ -1,104 +1,149 @@
     {
-        $model_comment = new Comment();
-        $model_dashboard = new Dashboard();
-        $model_user = new User();
+        $manager_comment = new CommentManager();
+        $comments = $manager_comment->get_comments();
         
-        $page="backend/dashboard";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+
+        //$manager_user = new Usermanager();
+        //$comments = $manager_user->
+        //$model_dashboard = new Dashboard();
+        //$model_user = new UserManager();
+
+       
+        if(filter_has_var(INPUT_GET, 'delete')){
+            $comment = $manager_comment->delete_comment();
+        }
+                
+        if(filter_has_var(INPUT_GET, 'approve')){
+            $comment = $manager_comment->approve_comment();
+        }
+        
+        require_once "config/dashboard.php";
+        $myView = new View('dashboard', 'backend');
+        $myView->render(array('comments' => $comments));
     }
 
     public function write()
     {
-        $model_article = new Article();
-        $model_user = new User();
+        $manager_article = new ArticleManager();
+        $errors = $manager_article->write_verification();
+         
+        //$model_user = new User();
         
-        $page="backend/write";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        //    if($model_user->admin()!=1){
+        //header("Location:/dashboard");
+    //}
+    
+        $myView = new View('write', 'backend');
+        $myView->render(array('errors' => $errors));
     }
 
     public function settings()
     {
-        $model_user = new User();
+        $manager_user = new UserManager();
+        $modos = $manager_user->get_modos();
+        $manager_user->settings_verification();
         
-        $page="backend/settings";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        //if($model_user->admin()!=1){
+        //header("Location:/dashboard");
+        //}
+        
+        $myView = new View('settings', 'backend');
+        $myView->render(array('modos' => $modos));
     }
 
     public function list()
     {
-        $model_article = new Article();
-        $model_user = new User();
+        $manager_article = new ArticleManager();
+        $posts = $manager_article->get_posts();
+        
+    
+        //$model_user = new User();
 
-        $page="backend/list";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        
+        //if($model_user->admin()!=1){
+        //header("Location:/dashboard");
+        //}
+        $myView = new View('list', 'backend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function logout()
     {
-        $model_user = new User();
-        
-        $page="backend/logout";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        unset($_SESSION['admin']);
+        header("Location:/articles-a-la-une");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)